web/client/selectors/plugins.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Check if an entry in localConfig is configured for a specific entry of `plugins`
+ * @param {string} entry the name of the entry
+ * @return {boolean} true if the entry exists, false otherwise
+ */
+export const isPluginsEntryConfigured = plugin => state => state.localConfig && state.localConfig.plugins && !!state.localConfig.plugins[plugin];
```

Comment:
I think the originally suggested isPluginConfigured name is better. It explains what the selector returns (is my plugin included in configuration?), not the technical details of entries in a JSON object.

Reply:
Yes I remember, but I though that is a page or a part of plugins entry that contains many plugins inside it. So even isPluginConfigured is a little obscure. (I expect to see if a single plugin is configured, not an entire page). I see also that the first argument as a misleading name. 

I can suggest 
- isPageConfigured pageName => ...
- isPluginsEntryConfigured entry => 
- isPluginsSetConfigured setName =>

Reply:
You are right, this is about a page, not a single plugin, hasPagePlugins? isPageConfigured is good too

Reply:
ok, done with isPageConfigured